src/Sulu/Bundle/WebsiteBundle/Admin/WebsiteAdmin.php

```diff
@@ -44,4 +67,22 @@ public function getJsBundleName()
     {
         return 'suluwebsite';
     }
+
+    /**
+     * Check the permissions for all webspaces.
+     * Returns true if one webspace has live permission.
+     *
+     * @return bool
+     */
+    private function checkLive()
+    {
+        foreach ($this->webspaceManager->getWebspaceCollection() as $webspace) {
+            $context = ContentAdmin::SECURITY_CONTEXT_PREFIX . $webspace->getKey();
+            if ($this->securityChecker->hasPermission($context, PermissionTypes::LIVE)) {
+                return true;
```

Comment:
From what I've understood the user needs this permission for all webspaces, because this action currently has an effect on all webspaces /cc @chirimoya